tests/functional/sync_v3_settings.js

```diff
@@ -74,7 +74,7 @@ define([
         .then(visibleByQSA('#delete-account .settings-unit-details'))
 
         .then(fillOutDeleteAccount(this, FIRST_PASSWORD))
-        .then(testIsBrowserNotified(this, 'delete_account'))
+        .then(testIsBrowserNotified(this, 'fxaccounts:delete_account'))
```

Comment:
Ran through this and noticed that `fxaaccounts:delete` is called as well. Not sure what the difference is, should be assert this was called as well?

```
1467732109096	FirefoxAccounts	DEBUG	FxAccountsWebChannel message received: fxaccounts:delete
1467732109096	FirefoxAccounts	DEBUG	FxAccountsWebChannel message received: fxaccounts:delete
1467732109099	FirefoxAccounts	DEBUG	FxAccountsWebChannel message received: fxaccounts:delete_account
1467732109099	FirefoxAccounts	DEBUG	FxAccountsWebChannel message received: fxaccounts:delete_account
```